codegen/datamodelgen.py

```diff
@@ -79,6 +78,19 @@ def _build_command_docstring(name: str, info: Any, indent: str):
     return doc
 
 
+def _build_query_docstring(name: str, info: Any, indent: str):
```

Comment:
This seems to be the same function as the one above aside from the first line.

Reply:
Yes, added check for command and query to handle this. Thanks.